app/javascript/src/components/SceneEditor.css

```diff
@@ -90,3 +91,13 @@
   font-size: 16px;
   padding: 9px 14px 6px;
 }
+
+.checkboxes label {
+  display: inline-block;
+  white-space: nowrap;
+}
+.checkboxes input {
+  box-shadow: none;
+  vertical-align: middle;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/16651533/78401300-6cf21400-75c6-11ea-94c5-f4ac8d7995f4.png)
@leobauza @efatsi This CSS styles the element in the picture above. It's is used in `SceneEditor.tsx` starting on line 170. The border/box you see displayed that the "M" is just barely passing through is the label's "border" attribute. The text is the text associated with the label, but for some reason it refuses to go in its own component's border. Here is what it looks like when commenting out the `white-space` line:

![image](https://user-images.githubusercontent.com/16651533/78401475-d07c4180-75c6-11ea-863b-ead9855980ff.png)

When you have a time, can you take a look at this and let me know if you have any ideas? The text is just as clickable for the checkbox as the border box itself, even when it's "outside" of the border.

Reply:
![great](https://i1.wp.com/css-tricks.com/wp-content/uploads/2017/06/css-is-awesome.jpg)

Reply:
If I knew more about CSS I could tell you what's going on, but I don't. And I think the FEDs are quite busy these days. But if this were me, I think you'd have better luck if you make the wrapping box it's own entity. Perhaps a new div with 100% width so it sits inline with the SETTINGS header box. And then the label/input shenanigans just have to worry about making the text appear next to the checkbox.

Reply:
@DanielSBrown we can pair on this quickly...hard to figure out from just looking at this here but I am sure 5-10 mins on a call we could sort it out I can make some time this week so hit me up on the slacks

Reply:
Will do, heads down on WCS right now but I'll have time later in the week